docs/bokeh/source/docs/reference.rst

```diff
@@ -49,3 +49,16 @@ be especially useful.
 :ref:`bokeh.settings`
     All Bokeh-related environment variables, which can be used to control
     things like resources, minification, and log levels, are documented here.
+
+
+Linking to the Bokeh documentation
+----------------------------------
+Developers of other packages that make use of Bokeh might want to use
+`intersphinx <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_
+to link to specific classes or functions in the Bokeh docs from their Sphinx
+documentation. The file needed for that is located at
+https://docs.bokeh.org/en/<version>/objects.inv
+where "<version>" can be the number of a released version or "latest", for example
+the entry in the ``conf.py`` file for Sphinx might look like this::
+
+    intersphinx_mapping = {'bokeh': ('https://docs.bokeh.org/en/latest/objects.inv ', None)}
```

Comment:
We don't really use implicit code blocks introduced with `::` anymore, prefer explicit `.. code-block:: python` for this